pom.xml

```diff
@@ -1747,7 +1747,7 @@
     <powermockVersion>2.0.9</powermockVersion>
     <okhttpVersion>3.10.0</okhttpVersion>
     <okioVersion>1.14.0</okioVersion>
-    <opennmsApiVersion>1.1.0-SNAPSHOT</opennmsApiVersion>
+    <opennmsApiVersion>2.0.0-SNAPSHOT</opennmsApiVersion>
```

Comment:
Can we avoiding bumping the major here?

Reply:
I think yes. The changes in [opennms-integration-api](https://github.com/OpenNMS/opennms-integration-api) were made in develop branch (2.0.0) ("Next" was specified in JIRA-Ticket). The changes there need to be picked from develop into release 1.x, then the version here can be also changed.

Reply:
Version was reduced